tests/integration/test_charm_ha.py

```diff
@@ -43,7 +43,7 @@ async def test_build_and_deploy(ops_test: OpsTest, loki_charm: str):
         # Secret must be at least 8 characters: https://github.com/canonical/minio-operator/issues/137
         ops_test.model.deploy(
             "minio",
-            channel="latest/stable",
+            channel="ckf-1.9/stable",
```

Comment:
Do we need to specify the track name? We could rely on the default, per today's discussion:
```
> juju deploy minio --channel=stable
Deployed "minio" from charm-hub charm "minio", revision 380 in channel ckf-1.9/stable on ubuntu@20.04/stable
```

Reply:
personally, I'd stay with `ckf-1.9/stable` for stability.  Its the same argument as pinning the python requirements - by pinning here, we know rerunning the test should get the same minio functionality every time.  Then if there's ever something we need from a newer minio, we bump

in practice for minio, which is essentially abandoned, it doesn't matter.  So I'm not against pinning to `stable`, I just think `ckf-1.9/stable` is the better general practice